src/Analytics/Adapter.php

```diff
@@ -98,13 +97,7 @@ public function setUserAgent(string $userAgent): self
      */
     public function createEvent(Event $event): bool
     {
-        try {
-            return $this->send($event);
-        } catch (\Exception $e) {
-            $this->logError($e);
-
-            return false;
-        }
+        return $this->send($event);
```

Comment:
We now throw exception on public function. Scarry, it may now start throwing in user-facing places, where we considered falures acceptable until now. Make sure to release new minor version and add bold red text informing about this.

Reply:
I think it's fine we should let the error handler on the server take care of it as 500 and log it on Sentry